24_find_lab/findlab.txt

```diff
@@ -1,24 +1,65 @@
-1. Write command lines to find using "find":
+1. Write command lines to find using "find":
+
 
   -. All files owned by yourself in /tmp folder
 
-  -. All files and directories not owned by you in /tmp folder
+  
+
+~/tmp find.
+
+-. All files and directories not owned by you in /tmp folder
+
+ 
+
+$ find . ! -user roi
+
+ -. All files, folders and links in your home folder whose name ends with a ".c"
 
-  -. All files, folders and links in your home folder whose name ends with a ".c"
+
+cd ~ find . -name '*.c'
+
 
   -. All files in your home folder whose name starts with an "a", "e", "i", "o" or "u"
 
-  -. All files in your home directory whose size is above 10MB
+  
+
+find . -name "*[aeiou]"

```

Comment:
The question asked about files starting with aeiou, while your command looks for aieou in the end. Better `find . -name '[aeiou]*'`